src/lib/bootstrap.php

```diff
@@ -19,6 +19,19 @@
 $loader = require __DIR__.'/../app/autoload.php';
 ZLoader::register($loader);
 
+// set current domain for using it as container parameter
+// http://symfony.com/doc/current/cookbook/configuration/external_parameters.html
+if (strpos(__DIR__, '/src/lib') !== false) {
+    $_SERVER['SYMFONY__ASSETS_BASE_URL'] = '//' . $_SERVER['HTTP_HOST'] . '/src/web';
+} else {
+    $_SERVER['SYMFONY__ASSETS_BASE_URL'] = '//' . $_SERVER['HTTP_HOST'] . '/web';
+}
+
```

Comment:
In general this works fine. The problem is the initial usage of Composer, like mentioned above.
After providing a default value Symfony does not accept the dynamic setting anymore.

I also tried this:
```
$assetsBaseUrl = '//' . $_SERVER['HTTP_HOST'] . ((strpos(__DIR__, '/src/lib') !== false) ? '/src/web' : '/web');
$core->getContainer()->setParameter('assets_base_url', $assetsBaseUrl);
```